### ISSUE TITLE
Error in coadd2d for X-SHOOTER NIR spectra

### ISSUE THREAD

**Issue by lukasfurtak** [pypeit/PypeIt#1332]
Hi,
I get the following error message for coadd2d:

```
[ERROR]   :: There is a mismatch between the number of valid orders found by PypeIt and the number expected for this spectrograph.  Unable to continue.  Please submit an issue on Github: https://github.com/pypeit/PypeIt/issues .
Traceback (most recent call last):
  File "/home/angers1NS/furtak/miniconda3/envs/spectroscopy/bin/pypeit_coadd_2dspec", line 11, in <module>
    sys.exit(CoAdd2DSpec.entry_point())
  File "/home/angers1NS/furtak/miniconda3/envs/spectroscopy/lib/python3.9/site-packages/pypeit/scripts/scriptbase.py", line 110, in entry_point
    cls.main(cls.parse_args())
  File "/home/angers1NS/furtak/miniconda3/envs/spectroscopy/lib/python3.9/site-packages/pypeit/scripts/coadd_2dspec.py", line 218, in main
    = coadd.reduce(pseudo_dict, show=args.show, show_peaks=args.peaks)
  File "/home/angers1NS/furtak/miniconda3/envs/spectroscopy/lib/python3.9/site-packages/pypeit/coadd2d.py", line 472, in reduce
    sobjs_obj, nobj, skymask_init = redux.find_objects(
  File "/home/angers1NS/furtak/miniconda3/envs/spectroscopy/lib/python3.9/site-packages/pypeit/reduce.py", line 582, in find_objects
    self.find_objects_pypeline(image,
  File "/home/angers1NS/furtak/miniconda3/envs/spectroscopy/lib/python3.9/site-packages/pypeit/reduce.py", line 1379, in find_objects_pypeline
    sobjs_ech, skymask[self.slitmask > -1] = extract.ech_objfind(
  File "/home/angers1NS/furtak/miniconda3/envs/spectroscopy/lib/python3.9/site-packages/pypeit/core/extract.py", line 1963, in ech_objfind
    msgs.error('There is a mismatch between the number of valid orders found by PypeIt and '
  File "/home/angers1NS/furtak/miniconda3/envs/spectroscopy/lib/python3.9/site-packages/pypeit/pypmsgs.py", line 184, in error
    raise PypeItError(msg)
pypeit.pypmsgs.PypeItError: There is a mismatch between the number of valid orders found by PypeIt and the number expected for this spectrograph.  Unable to continue.  Please submit an issue on Github: https://github.com/pypeit/PypeIt/issues .

```

It comes for simply trying to coadd 2 telluric standard spectra. Spectra and config file are attached, thanks!

[Telluric2_coadd2d.txt](https://github.com/pypeit/PypeIt/files/8022736/Telluric2_coadd2d.txt)
[spec1d_XSHOO.2021-11-15T00:03:26.550-STD,TELLURIC_XShooter_NIR_20211115T000326.550.txt](https://github.com/pypeit/PypeIt/files/8022737/spec1d_XSHOO.2021-11-15T00.03.26.550-STD.TELLURIC_XShooter_NIR_20211115T000326.550.txt)
[spec1d_XSHOO.2021-11-15T00:04:24.835-STD,TELLURIC_XShooter_NIR_20211115T000424.835.txt](https://github.com/pypeit/PypeIt/files/8022738/spec1d_XSHOO.2021-11-15T00.04.24.835-STD.TELLURIC_XShooter_NIR_20211115T000424.835.txt)

**Comment by jhennawi** [comment]
Hi @lukasfurtak, 

My guess is that some orders got masked. You can check this by running pypeit_show_2dspec and inspecting the original frames. It will probably be easier to get assistance via the PypeIt Users Slack channel. Also, while this does not speak to any issues you are encountering, I don't think it makes much sense to coadd telluric standards in 2d. A 1d coadd is surely the way to go there.

**Comment by lukasfurtak** [comment]
Hi @jhennawi,

thanks for your answer. Could I have a link to the slack channel then?

I agree about the co-adding in 2d, this was a test run because I got some issue 'no object found' when co-adding science frames in 2d so I tried co-adding something that definitely has a detection.

**Comment by kbwestfall** [comment]
The user's slack invite is in the closed issue #676 .  Please request a new invite link if the current one is outdated.

Closing.  Please re-open if this is still an issue.
